CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Obl.ong
+
+First of all, thank you for wanting to contribute to Obl.ong! Everyone is welcome to help and encouraged to help.
+
+## 📜 Code of Conduct
+
+Obl.ong is governed by the [Code of Conduct](https://github.com/obl-ong/code-of-conduct). You are expected to follow it. Basically, treat other humans with respect and don't discriminate based on any characteristic. You can report any violation to team@obl.ong, and we will take care of it ASAP.
+
+## 🪳 Bug Reporting
+
+Bug Reports are vital to development. You can report bugs using GitHub issues. However, we need information to help squash the bug. Basically:
+
+- Ensure you are using the latest commit or version of Obl.ong.
+- Make sure the issue hasn't already been posted by searching the issues.
+- Give information about the bug: OS, OS version, Browser Version (if it occured in the browser), ruby version (`ruby -v`), git commit (`git rev-parse --short HEAD`), screenshot (if needed).
+
+Also, please do **not** submit security issues via GitHub issues. Please send an E-mail to team@obl.ong.
+
+## 💻 Setting up a development environment
+
+1. Install the currently supported version Ruby. You can use a tool like [Ruby Version Manager (RVM)](https://rvm.io/) to install ruby. Check the `.ruby-version` file to find it. Usually, it's the latest version, but it never hurts to check.
```

Comment:
rbenv > rvm

Reply:
Fixed, but what's the difference? I don't know too much about ruby, sorry

Reply:
rbenv is just a better tool that I will actually support if people have questions.